backend/src/main/kotlin/com/example/demo/controller/dto/Card.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.controller.dto
+
+import java.io.Serializable
+import com.example.demo.model.Card
+
+data class CardDTO (
+    private val id: Long,
+    private val name: String,
+    private val playerNumber: Short,
+    private val photoURL: String?,
+    private val playerPosition: String,
+    private val country: String
+):Serializable {
+    fun getId(): Long {
```

Comment:
you should not need getters and setters on kotlin, that is a java practice. Don't delcare the elements as private